src/InPluginUpsells/resources/js/sale-banner.js

```diff
@@ -0,0 +1,36 @@
+const bodyIsReady = onReady => window.requestAnimationFrame( document.body ? onReady : bodyIsReady );
```

Comment:
Actually, we don't even need this because this script is loaded in page footer 😅

Reply:
Resolved in eb5de4e